aspnetcore/fundamentals/logging/index.md

```diff
@@ -34,7 +34,7 @@ The default ASP.NET Core web app templates:
 
 [!code-csharp[](index/samples/6.x/WebApp/Program.cs?name=snippet1&highlight=1)]
 
-The preceding code shows the *Program.cs* file  created with the ASP.NET Core web app templates. The next several sections provide samples based on the ASP.NET Core web app templates, which use the Generic Host. [Non-host console apps](#nhca) are discussed later in this document.
+The preceding code shows the *Program.cs* file created with the ASP.NET Core web app templates. The next several sections provide samples based on the ASP.NET Core web app templates, which use the Generic Host.
```

Comment:
They don't really use Generic Host. They use the new minimal hosting model which uses most of the same code as Generic Host.

@serpent5  should we create an issue to get a consistent story on this across docs. I think we have a bunch of docs that mention the same thing.

Reply:
Yeah, I think so. Now we know the minimal APIs deal is being described as a new hosting model, we should be able to do something about this problem.